### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by Anchaliya75** [danielgatis/rembg#469] [Q&A]
While calling remove function it is giving error 
Retrieve() got an unexpected keyword argument  'progressbar'
How to resolve this?

Using rembg 2.0.43 version

**Comment by tenghuiji** [comment]
This issue is caused by a change in the pooch.retrieve method parameter. The problem can be solved by commenting out this parameter.    

u2net.py
@classmethod
    def download_models(cls, *args, **kwargs):
        fname = f"{cls.name(*args, **kwargs)}.onnx"
        pooch.retrieve(
            "https://github.com/danielgatis/rembg/releases/download/v0.0.0/u2net.onnx",
            None
            if cls.checksum_disabled(*args, **kwargs)
            else "md5:60024c5c889badc19c04ad937298a77b",
            fname=fname,
            path=cls.u2net_home(*args, **kwargs),
            #progressbar=True,
        )
